core/src/main/java/com/yahoo/oak/OakUnsafeDirectBuffer.java

```diff
@@ -32,9 +32,6 @@
  * }
  * }
  * </pre>
- * <p>
- * Note: in the above example, the following will throw a ReadOnlyBufferException because the buffer mode is read-only:
```

Comment:
Again do not see where is the write attempt... Just curious! :)

Reply:
Again, below (line 37).